app_flutter/lib/task_box.dart

```diff
@@ -29,10 +30,26 @@ class TaskBox extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    if (task.status == 'In Progress') {
+      return Container(
+        margin: const EdgeInsets.all(1.0),
+        color: statusColor['New'],
```

Comment:
With some careful planning this whole widget could be const

Reply:
I simplified some of the code reuse, but right now TaskBox can't be made const because of the mapping function to get a Task in StatusGrid isn't const. 
Do you have any recommendations on what to try to make the widget const?